README.md

```diff
@@ -39,6 +42,17 @@ But these third-parties are not required as a prerequisite for runtime.
 Tracking issues, reporting bugs and contributing to the codebase and documentation are on GitHub at:
 https://github.com/enthought/comtypes
 
+<a id="ongoing-plans"></a>
+### Ongoing plans
+For the time being, the development target branch of this package will be the [`drop_py2` branch](https://github.com/enthought/comtypes/tree/drop_py2) and the [`master` branch](https://github.com/enthought/comtypes/tree/master) will be in maintenance-only-mode.
+
+As the name suggests, `drop_py2` is a mid-term-planning branch to drop supporting `Python` 2.x from this package, and start supporting `Python` 3.x only.
+
+The `drop_py2` branch will be merged into the `master` branch in the future, and the `master` branch will back to the development target branch.
```

Comment:
I'd suggest to mention future renaming `master` branch to `main` along with merging `drop_py2`.

Reply:
From your suggestion, I have added that.
Please review this.